### PR TITLE
Update job event descriptions

### PR DESCRIPTION
**Pull request by harrietgrace**
We've been using the descriptions from the main bk app's config yml in here for a while. They don't really describe what's actually going on, and we've had a few people now ask about what the difference is between started and activated. 

I couldn't find if those snippets are used elsewhere in the app, so I decided to just change them in the docs 👍🏻

### CODE REVIEW

File: pages/apis/webhooks/job_events.md.erb [hunk 1]
```diff
@@ -8,15 +8,15 @@
 <tbody>
   <tr>
     <th><code>job.started</code></th>
-    <td><%= webhook_description "job.started" %></td>
+    <td>A command step has been started</td>
```
Comment (toolmantim) [inline pages/apis/webhooks/job_events.md.erb]: What about "A command step job has been started"? You can't start a step, per-se. But jobs have "steps" which are like templates. Trick!

Comment (toolmantim) [reply pages/apis/webhooks/job_events.md.erb]: Also, perhaps its "A command step job has started running on an agent"? I just checked the code, and that seems to be the case.

File: pages/apis/webhooks/job_events.md.erb [hunk 2]
```diff
@@ -8,15 +8,15 @@
...
   </tr>
   <tr>
     <th><code>job.finished</code></th>
-    <td><%= webhook_description "job.finished" %></td>
+    <td>A job has been finished</td>
   </tr>
   <tr>
     <th><code>job.activated</code></th>
-    <td><%= webhook_description "job.activated" %></td>
+    <td>A block step has been unblocked</td>
```
Comment (toolmantim) [inline pages/apis/webhooks/job_events.md.erb]: "A block step job has been unblocked" perhaps?

File: pages/apis/webhooks/job_events.md.erb (hunk 1 above)
Comment (harrietgrace) [reply pages/apis/webhooks/job_events.md.erb]: Hah I had 'job' in my first draft, then took it out 😄 Happy to add it back in!

Comment (toolmantim) [reply pages/apis/webhooks/job_events.md.erb]: Hah, cool! Yeah, I reckon let’s add it.

**Comment by harrietgrace** [comment]
Reverted back to using the variables, and started using the variables in the main Webhooks overview document too. Actual changes to the descriptions are now in https://github.com/buildkite/buildkite/pull/3473.

**Pull request merged**